Documentation/cmdref/cilium-agent.md

```diff
@@ -80,6 +80,7 @@ cilium-agent [flags]
       --enable-k8s-event-handover                     Enable k8s event handover to kvstore for improved scalability
       --enable-l7-proxy                               Enable L7 proxy for L7 policy enforcement (default true)
       --enable-local-node-route                       Enable installation of the route which points the allocation prefix of the local node (default true)
+      --enable-monitor-socket                         Enable the monitor unix domain socket server (default true)
```

Comment:
Yeah I was torn a bit about the naming (since with `--enable-monitor=false`, the monitor agent still remains active for Hubble). But that from the user's perspective, I guess this should not matter. I'll rename it.